### DISCUSSION TITLE
[UTM Documentation] Windows 10 goes to Windows 11

### DISCUSSION THREAD

**Discussion by lorenzoalali** [utmapp/UTM#6273] [General]
On https://docs.getutm.app/guides/guides/
the link to Windows 10 (https://docs.getutm.app/guides/windows-10/)
redirects to the installation guide of Windows 11
(listed as https://docs.getutm.app/guides/windows/)
instead of to the Windows 10 one.
The guide for Windows 10 is nowhere to be found.

<img width="677" alt="Capture d’écran 2024-04-13 à 21 42 07" src="https://github.com/utmapp/UTM/assets/11087992/0d6195b5-575f-4b28-b5f6-e10c0008e964">

**Comment by osy** [comment]
Yes this is intentional. As Microsoft has stopped providing windows 10 images there's no legal way to install it anymore.

**Reply by lorenzoalali** [reply]
Thank you @osy, in that case, may I then suggest
- either redirecting this Windows 10 link to a page that explains what you wrote
- either delete the Windows 10 link altogether from that panel?

**Reply by step21** [reply]
I would second this, it is super confusing. Plenty of people still have images, and are you sure microsoft has stopped providing images? for me, crystalfetch worked perfectly fine to download it. I can also prepare a PR if that helps.
